### DISCUSSION TITLE
Cloud Support

### DISCUSSION THREAD

**Discussion by fayt020** [MikeMaximus/gbm#267] [General]
Are there any plans to implement cloud support? I have a SFTP server that would work perfect with this program.

**Comment by MikeMaximus** [comment]
I don't have plans to support any specific cloud service or technology.

If you're on Windows, I can recommend using [GoodSync](https://www.goodsync.com/) along with GBM.  I've used it in the past and it supports folder syncing with SFTP.

**Reply by fayt020** [reply]
It's unfortunate that I can't map the directory from my save files to my server without additional software. Thank you for the reply and the program.
